app/bin/service/analyzer.dart

```diff
@@ -21,15 +21,14 @@ import 'package:pub_dartlang_org/analyzer/handlers.dart';
 import 'package:pub_dartlang_org/analyzer/pana_runner.dart';
 import 'package:pub_dartlang_org/analyzer/task_sources.dart';
 
-final Logger logger = new Logger('pub.analyzer');
+final Logger _logger = new Logger('pub.analyzer');
 
 Future main() async {
   useLoggingPackageAdaptor();
 
   withAppEngineServices(() async {
-    _initFlutterSdk().then((_) async {
-      startIsolates(logger, _runSchedulerWrapper);
-    });
+    // TODO(kevmoo): check that the flutter versions align
```

Comment:
Which flutter version needs to be in sync? Should this comment be addressed before merging the PR?

I thought we had the git hash in some shell script or something.

Reply:
We could check that the environment variable is set correctly – and aligns w/ what's in the Dart source...